src/Rules/Functions/CallUserFuncArrayRule.php

```diff
@@ -0,0 +1,81 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Functions;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\ArgumentsNormalizer;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\ReflectionProvider;
+use PHPStan\Rules\FunctionCallParametersCheck;
+use PHPStan\Rules\Rule;
+use function count;
+use function ucfirst;
+
+/**
+ * @implements Rule<FuncCall>
+ */
+class CallUserFuncArrayRule implements Rule
```

Comment:
should this rule here also cover https://github.com/phpstan/phpstan/issues/5934 - or do we need a separate rule for that?